java/bogdana_maksiuta/src/com/zoolatech/lecture1/tasks/_10/GuessANumberGame.java

```diff
@@ -0,0 +1,66 @@
+package com.zoolatech.lecture1.tasks._10;
+
+import java.util.Random;
+import java.util.Scanner;
+
+/**
+ * 10. Write a program that simulates the guess a number game.
+ * The program should generate a random number between 1 and N (N should be read from the input)
+ * and the user needs to guess it. If the user enters a number less than guessed,
+ * the program needs to output “Larger”. If the user enters a number greater than guessed -
+ * the program should output “Smaller”. If the user guesses the number correctly -
+ * the program should output “That’s it!” and finish.
+ * Input: 10
+ * Input: 7
+ * Output: Smaller
+ * Input: 3
+ * Output: Larger
+ * Input: 5
+ * Output: That’s it!
+ */
+public class GuessANumberGame {
+    static Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        config();
+    }
+
+    public static void config () {
+        int maxLimit = 0;
+        while (maxLimit < 5 || maxLimit > 500) {
```

Comment:
Why have you chosen such boundaries for the max value - 5 and 500? The task description doesn't say anything regarding the boundaries of the max value.

Reply:
it's was easy for me to test my code. fixed